jupyter-js-widgets/package.json

```diff
@@ -47,6 +47,7 @@
     "npm-run-all": "^1.5.1",
     "rimraf": "^2.4.1",
     "sinon": "^1.17.2",
+    "sinon-chai": "^2.8.0",
```

Comment:
I got an error running the tests, saying it couldn't find that module. After including it as dev-dependency, tests ran.